src/pybind/mgr/cephadm/services/monitoring.py

```diff
@@ -583,6 +608,16 @@ def ok_to_stop(self,
             return HandleCommandResult(-errno.EBUSY, '', warn_message)
         return HandleCommandResult(0, warn_message, '')
 
+    def find_prometheus_credentials(self, multicluster_config, url):
+        for cluster_id, clusters in multicluster_config['config'].items():
```

Comment:
`cluster_id` is not used you can replace it by _

Reply:
Updated the PR